entry_types/scrolled/package/src/frontend/MediaPlayer/index.js

```diff
@@ -0,0 +1,26 @@
+import {media} from 'pageflow/frontend';
+
+export class MediaPlayer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  componentDidMount(){    
+    if (this.props.sources) {
+      let player = media.getPlayer(this.props.sources, {
+        type: this.props.type,
+        poster: this.props.poster
+      });
+      this.setState({player: player});
+    }
+  }
+  render(){
+    let playerHTML = '';
+    if (this.state.player) {
+      playerHTML = this.state.player.el().innerHTML;
+    }
+    return (
+      <div dangerouslySetInnerHTML={{__html: playerHTML}}></div>
+    );
+  }
+}
```

Comment:
Let's use hooks for all new components.

Using `innerHTML` basically recreates the DOM elements. This will neither work with blessed elements. Also it causes the inserted media elements to no longer be controlled by the player instance created above. We can use a ref and `appendChild` instead.